tests/benchmarks/data_structure_perf/dlist_perf/src/dlist_perf.c

```diff
@@ -173,12 +173,96 @@ void test_dlist_for_each(void)
 	zassert_true(count == 2, "SYS_DLIST_ITERATE_FROM_NODE failed");
 }
 
+/**
+ * @brief Test that access the 'head' and 'tail' in constant time
+ *
+ * @details Defined a double list and append several nodes.
+ * defined two pointers---'head','tail'.No matter how many nodes
+ * dlist have, get head and tail from the dlist directly.the time
+ * complexity of accessing head and tail is O(1).
+ *
+ * @ingroup lib_dlist_tests
+ */
+void test_dlist_peak_head_tail(void)
+{
+	sys_dlist_t list;
+	sys_dnode_t node[10];
+
+	sys_dlist_init(&list);
+
+	for (int i = 0; i < ARRAY_SIZE(node); i++) {
+		sys_dlist_append(&list, &node[i]);
+	}
+
+	sys_dlist_t *head = NULL, *tail = NULL;
```

Comment:
sorry for not having said about it. This variables don't need to be initialized and they can be declared together with the others in this function.

Reply:
Good to know! Thanks @ceolin , Would you approve this PR if there are not anything wrong? :)